src/urllib3/connection.py

```diff
@@ -743,3 +801,13 @@ class DummyConnection:
 
 
 VerifiedHTTPSConnection = HTTPSConnection
+
+
+def url_from_connection(
```

Comment:
Since the `url_from_connection` and `url_from_pool` functions aren't meant to be a part of our public API we should make them private via an underscore prefix.

Reply:
done